src/platform/plugins/shared/workflows_management/public/features/workflow_execution_list/ui/workflow_execution_list_item.tsx

```diff
@@ -27,103 +27,101 @@ import { getStatusLabel } from '../../../shared/translations';
-              {startedAt ? (
-                <EuiToolTip position="right" content={formattedDate}>
-                  <FormattedRelative value={startedAt} />
-                </EuiToolTip>
-              ) : (
-                <FormattedMessage
-                  id="workflows.workflowExecutionListItem.notStarted"
-                  defaultMessage="Not started"
-                />
-              )}
-            </EuiFlexItem>
-          </EuiFlexGroup>
-        </EuiFlexItem>
-        <EuiFlexItem>
-          <EuiFlexGroup
-            alignItems="center"
-            justifyContent="flexEnd"
-            gutterSize="xs"
-            responsive={false}
-          >
-            <EuiFlexItem grow={false}>
-              <span css={styles.duration}>{formattedDuration}</span>
-            </EuiFlexItem>
-            {onClick ? (
+    return (
+      <EuiPanel color="plain" hasShadow={false} paddingSize="m" hasBorder css={panelCss}>
+        <EuiFlexGroup
+          gutterSize="m"
+          alignItems="center"
+          justifyContent="flexStart"
+          onClick={onClick ?? undefined}
+          responsive={false}
+          color="plain"
+        >
+          <EuiFlexItem grow={false}>{getExecutionStatusIcon(euiTheme, status)}</EuiFlexItem>
+          <EuiFlexItem>
+            <EuiFlexGroup direction="column" gutterSize="xs">
+              <EuiFlexItem>
+                <EuiText
+                  size="s"
+                  css={{ fontWeight: 'bold', color: getExecutionTitleColor(euiTheme, status) }}
+                >
+                  {getStatusLabel(status)}
+                </EuiText>
+              </EuiFlexItem>
+              <EuiFlexItem>
+                {startedAt ? (
+                  <EuiToolTip position="left" content={formattedDate}>
+                    <EuiText size="xs" tabIndex={0} color="subdued">
+                      <FormattedRelative value={startedAt} />
+                    </EuiText>
+                  </EuiToolTip>
+                ) : (
+                  <EuiText size="xs" color="subdued">
+                    <FormattedMessage
+                      id="workflows.workflowExecutionListItem.notStarted"
+                      defaultMessage="Not started"
+                    />
+                  </EuiText>
+                )}
+              </EuiFlexItem>
+            </EuiFlexGroup>
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiFlexGroup alignItems="center" justifyContent="flexEnd" gutterSize="xs" wrap>
               <EuiFlexItem grow={false}>
-                <EuiIcon type="arrowRight" color={euiTheme.colors.backgroundFilledText} />
+                <EuiIcon type="clock" color="subdued" />
               </EuiFlexItem>
-            ) : null}
-          </EuiFlexGroup>
-        </EuiFlexItem>
-      </EuiFlexGroup>
-    </EuiPanel>
-  );
-};
+              <EuiFlexItem grow={false}>
+                <EuiText size="xs" color="subdued">
```

Comment:
@r4zr32d3k1l asked for 12.25px font-size, which is 0.875 of 14px, according to euiTheme it's 's' size, but for some reason it gives 12px. 

<img width="807" height="581" alt="Image" src="https://github.com/user-attachments/assets/3616619a-7b96-459e-b788-a3ce6cd4d4e9" />
<img width="1016" height="601" alt="Image" src="https://github.com/user-attachments/assets/17de2223-b2ef-413d-9a8a-3f7a75fb0fdc" />